frontend/package.json

```diff
@@ -7,6 +7,9 @@
     "name": "John Doe",
     "company": "ACME, Inc."
   },
+  "scripts": {
```

Comment:
Upss I forgot this, I thought including this `script`, the people would not have to install globaly `lineman`